src/make_data/fake-data.R

```diff
@@ -249,33 +248,32 @@ write.csv(x = master, file = here("data/fake-data/master.csv"), quote = TRUE, ro
 # Column names -----------------------------------------------------------------
 
 nhs_column_names <-
-  c("nhsnumber",
-    "dateofbirth",
-    "patientfirstname",
-    "patientothername",
-    "patientsurname",
-    "patientaddress_line1",
-    "patientaddress_line2",
-    "patientaddress_line3",
-    "patientaddress_line4",
-    "patientaddress_line5",
-    "patientaddress_postcode",
-    "gppracticecode",
-    "practice_name",
+  c("Traced_NHSNUMBER",
+    "DateOfBirth",
+    "PatientFirstName",
+    "PatientOtherName",
+    "PatientSurname",
+    "PatientAddress_Line1",
+    "PatientAddress_Line2",
+    "PatientAddress_Line3",
+    "PatientAddress_Line4",
+    "PatientAddress_Line5",
+    "PatientAddress_Postcode",
+    "GPPractice_Code",
+    "Practice_NAME",
     "contact_telephone",
     "mobile",
     "patient_landline",
```

Comment:
Should this change to just `landline` as per requirements?

```suggestion
    landline = ch_phone_number(n, locale = "en_GB"),
```

Reply:
Good spot, thanks.